.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+<!--
+Thank you for your pull request and contribution to hyoka. 
+
+Contributors guide: ./CONTRIBUTING.md
+-->
+
+## Proposed Changes
+
+<!-- Please provide a complete list of the proposed changes here. -->
+
+-
+-
+
+## Notes
+
+<!-- Please provide additional notes here (if applicable). -->
+
+## Closing issues
+
+Put `Closes #XXXX` in your comment to auto-close the issue that your PR fixes (if applicable).
+
```

Comment:
this part should be  a comment at the top something like
```md
<!-- If your PR fixes an open issue, use `Closes #101` to link your PR with the issue. #101 stands for the issue number you are fixing -->
```

Reply:
Hi, ofc I'll change, but do you mean at the top of Closing Issues part? Or do you want me to remove Closing Issues and add this comment at the top of the whole template? Thanks

Reply:
yeah, I think it's better to remove the closing issues section and have that comment at the top instead. Thanks.

Reply:
@olamide203 just added the change :)